Cppcheck errors reported in file_offload.c

**Describe the bug**
Initial manual run of cppcheck reports the following concerns:

```
Checking store/file_offload.c ...
store/file_offload.c:512:101: style: inconclusive: Function 'bplib_file_offload_restore' argument 3 names different: declaration 'pblk' definition 'pblk_out'. [funcArgNamesDifferent]
static int bplib_file_offload_restore(bplib_mpool_block_t *svc, bp_sid_t sid, bplib_mpool_block_t **pblk_out)
                                                                                                    ^
store/file_offload.c:48:101: note: Function 'bplib_file_offload_restore' argument 3 names different: declaration 'pblk' definition 'pblk_out'.
static int bplib_file_offload_restore(bplib_mpool_block_t *svc, bp_sid_t sid, bplib_mpool_block_t **pblk);
                                                                                                    ^
store/file_offload.c:512:101: note: Function 'bplib_file_offload_restore' argument 3 names different: declaration 'pblk' definition 'pblk_out'.
static int bplib_file_offload_restore(bplib_mpool_block_t *svc, bp_sid_t sid, bplib_mpool_block_t **pblk_out)
                                                                                                    ^
store/file_offload.c:372:9: error: Uninitialized variable: eblk [uninitvar]
    if (eblk != NULL)
        ^
store/file_offload.c:372:9: error: Uninitialized variable: eblk [uninitvar]
    if (eblk != NULL)
        ^
store/file_offload.c:342:21: note: Assuming condition is false
    if (read_status == BP_SUCCESS)
                    ^
store/file_offload.c:372:9: note: Uninitialized variable: eblk
    if (eblk != NULL)
        ^
store/file_offload.c:190:20: style: Variable 'result' is assigned a value that is never used. [unreadVariable]
            result = mkdir(name_buf, 0755);
                   ^
store/file_offload.c:196:20: style: Variable 'result' is assigned a value that is never used. [unreadVariable]
            result = mkdir(name_buf, 0755);
                   ^
```

**To Reproduce**
Execute cppcheck against source code

**Expected behavior**
Should execute cleanly with no errors reported

**System observed on:**
Ubuntu 22.04

**Reporter Info**
Joseph Hickey, Vantage Systems, Inc.

--- a/store/file_offload.c
+++ b/store/file_offload.c
@@ -45,7 +45,7 @@ static int bplib_file_offload_query(bplib_mpool_block_t *svc, int key, bplib_cac
 static int bplib_file_offload_start(bplib_mpool_block_t *svc);
 static int bplib_file_offload_stop(bplib_mpool_block_t *svc);
 static int bplib_file_offload_offload(bplib_mpool_block_t *svc, bp_sid_t *sid, bplib_mpool_block_t *pblk);
-static int bplib_file_offload_restore(bplib_mpool_block_t *svc, bp_sid_t sid, bplib_mpool_block_t **pblk);
+static int bplib_file_offload_restore(bplib_mpool_block_t *svc, bp_sid_t sid, bplib_mpool_block_t **pblk_out);
 static int bplib_file_offload_release(bplib_mpool_block_t *svc, bp_sid_t sid);
 
 typedef struct bplib_file_offload_state
@@ -188,13 +188,21 @@ static void bplib_file_offload_sid_to_name(bplib_file_offload_state_t *state, ch
         {
             *pd2   = 0;
             result = mkdir(name_buf, 0755);
-            *pd2   = '/';
+            if (result != 0)
+            {
+                bplog(NULL, BP_FLAG_DIAGNOSTIC, "mkdir(%s): %s\n", name_buf, strerror(errno));
+            }
+            *pd2 = '/';
         }
         if (pd1 != NULL)
         {
             *pd1   = 0;
             result = mkdir(name_buf, 0755);
-            *pd1   = '/';
+            if (result != 0)
+            {
+                bplog(NULL, BP_FLAG_DIAGNOSTIC, "mkdir(%s): %s\n", name_buf, strerror(errno));
+            }
+            *pd1 = '/';
         }
     }
 }
@@ -322,6 +330,8 @@ static int bplib_file_offload_read_payload(int fd, bplib_file_offload_record_t *
     bplib_mpool_block_t *eblk;
     size_t               chunk_sz;
     int                  read_status;
+
+    eblk = NULL;
 
     /* payload block: size and offset info written in native form, followed by encoded CBOR data */
 

